puma_joy:
  joy_topic: "joy"
  min_accel: 0
  max_accel: 40 # max value 126

  angle_min_degree: -45
  angle_max_degree: 45